src/ctype/CType.utils.ts

```diff
@@ -107,7 +108,13 @@ export function decompressSchema(
  */
 
 export function compress(cType: ICType): CompressedCType {
-  if (!cType.hash || !cType.owner || !cType.schema) {
+  if (
+    !cType.hash ||
+    (typeof cType.owner === 'string'
+      ? !checkAddress(cType.owner, 42)[0]
```

Comment:
Is `42` the length of the address or the maximum number it can be?

Reply:
as per https://github.com/paritytech/substrate/wiki/External-Address-Format-(SS58): 00101010b (42) Generic Substrate wildcard (SS58 checksum preimage).

This this is basically the address prefix, which is ofc required parameter.